scripts/release/README.md

```diff
@@ -0,0 +1,18 @@
+# Enceladus Release Scripts
+
+## release_prepare
+
+`release_prepare.sh` / `release_prepare.cmd`
+
+* Merges the current _develop_ branch into the _master_ branch, while removing the `_SNAPSHOT` suffix from the modules' version.
```

Comment:
```suggestion
* Removes the `-SNAPSHOT` version suffix in all submodules in the current _develop_ branch and merges it into the _master_ branch.
```

Because this better communicates the order in which the actions are happening.